lib/pc_patch_uncompressed.c

```diff
@@ -215,6 +216,8 @@ pc_patch_uncompressed_compute_extent(PCPATCH_UNCOMPRESSED *patch)
 void
 pc_patch_uncompressed_free(PCPATCH_UNCOMPRESSED *patch)
 {
+	pc_patch_free_stats((PCPATCH*) patch);
+
```

Comment:
As you removed the check in `pc_patch_free_stats`, maybe you can add `assert(patch);` here, just to be consistent across patch types

Reply:
Good point. I'll add that.

Nice picture by the way ;)

Reply:
New commit added.